packages/smooth_app/lib/helpers/extension_on_text_helper.dart

```diff
@@ -29,3 +29,18 @@ extension Selectable on Text {
           );
   }
 }
+
+extension StringExtensions on String {
+  int count(String character) {
+    assert(character.length == 1);
+
+    int count = 0;
+    for (int i = 0; i != length; i++) {
```

Comment:
```suggestion
    for (int i = 0; i < length; i++) {
```
Tbh that's the first time I see a `!=` test rather than a `<` test for an array loop, in 41 years of programming.
Technically that will do the same, but I would recommend the standard `<` syntax.